src/WebJobs.Script/Host/FunctionMetadataProvider.cs

```diff
@@ -80,41 +86,44 @@ private Collection<FunctionMetadata> ReadFunctionsMetadata()
 
         private FunctionMetadata ReadFunctionMetadata(string functionDirectory, IEnumerable<WorkerConfig> workerConfigs, Dictionary<string, ICollection<string>> functionErrors, IFileSystem fileSystem = null)
         {
```

Comment:
same here, is it useful to have breakdown at the methodlevel instead of just ReadFunctionsMetadata?

Reply:
Each ReadFunctionMetadata is a file read operation. THought it would be useful to have an overall metric (ReadFunctionsMetadata) and a breakdown as well.

---
In reply to: [333799507](https://github.com/Azure/azure-functions-host/pull/4973#discussion_r333799507) [](ancestors = 333799507)